include/rtc/rtc.h

```diff
@@ -77,6 +77,28 @@ typedef enum { // Don't change, it must match plog severity
 	RTC_LOG_VERBOSE = 6
 } rtcLogLevel;
 
+#if RTC_ENABLE_MEDIA
+
+typedef enum {
+    // video
+    RTC_H264,
+    RTC_VP8,
+    RTC_VP9,
+
+    // audio
+    RTC_OPUS
+} rtcCodec;
+
+typedef enum {
+    RTC_SENDONLY,
```

Comment:
Same, this should be `RTC_DIRECTION_SENDONLY` for consistency.

Reply:
Done